packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -270,8 +305,14 @@ myTooltipEl.addEventListener('hidden.tooltip', () => {
 tooltip.hide();
 ```
 
-[example]: https://spirit-design-system-demo.netlify.app/src/scss/components/tooltip/#advanced-positioning
 [dictionary-placement]: https://github.com/lmc-eu/spirit-design-system/tree/main/docs/DICTIONARIES.md#placement
+[example]: https://spirit-design-system-demo.netlify.app/src/scss/components/tooltip/#advanced-positioning
+[floating-ui-flip-crossaxis]: https://floating-ui.com/docs/flip#crossaxis
+[floating-ui-flip-fallbackaxissidedirection]: https://floating-ui.com/docs/flip#fallbackaxissidedirection
+[floating-ui-flip-fallbackplacements]: https://floating-ui.com/docs/flip#fallbackplacements
```

Comment:
I'd rather stick to hyphenation even in the parameter names, it's much more readable:

```suggestion
[floating-ui-flip-cross-axis]: https://floating-ui.com/docs/flip#crossaxis
[floating-ui-flip-fallback-axis-side-direction]: https://floating-ui.com/docs/flip#fallbackaxissidedirection
[floating-ui-flip-fallback-placements]: https://floating-ui.com/docs/flip#fallbackplacements
```

Reply:
hyphenation added for better readabilty